### DISCUSSION TITLE
Popup Edit

### DISCUSSION THREAD

**Discussion by DelScipio** [zauberzeug/nicegui#1125] [Ideas / Feature Requests]
Quasar has this component https://quasar.dev/vue-components/popup-edit

This could be very interesting as a easy way to edit text and complemente the WYSIWYG editor component if implemeneted. Other frameworks have this component and is useful.

**Comment by falkoschindler** [comment]
That would be a great addition to NiceGUI's UI elements. Anyone up for giving it a try? A pull request would be very welcome.

**Comment by falkoschindler** [comment]
I just played around with QPopupEdit and the generic `ui.element` and it works pretty nicely:
```py
with ui.label('Hello, world!') as label:
    with ui.element('q-popup-edit'):
        ui.input().bind_value(label, 'text')
```

**Reply by v479038280** [reply]
```py
bind_value(
    target_object: Any,
    target_name: str = 'value',
    *,
    forward: (...) -> Any = lambda x: x,
    backward: (...) -> Any = lambda x: x
)
```

Where can we find the `target_name` that we can use?

**Reply by falkoschindler** [reply]
@v479038280 You can basically use any attribute of the `target_object`. Since the label has a `text` property, we can use `'text'` here.

**Comment by evnchn** [comment]
Sorry. I gave it a shot. However, the element is likely non-workable until we resolve https://github.com/zauberzeug/nicegui/issues/2149
